public/app/plugins/panel/geomap/layers/data/dayNightLayer.tsx

```diff
@@ -162,41 +164,43 @@ export const dayNightLayer: MapLayerRegistryItem<DayNightConfig> = {
     // Crosshair sharing subscriptions
     const subscriptions = new Subscription();
 
-    subscriptions.add(
-      eventBus.subscribe(DataHoverEvent, (event) => {
-        console.log('EVENT', {...event.payload.point} );
-        const time = event.payload?.point?.time as number;
-        if (time) {
-          const lineTime = new Date(time);
-          const nightLinePoints = sourceLine.getCoordinates(lineTime.toString(), 'line');
-          nightLineLayer.getSource()?.clear();
-          const lineStringArray:Coordinate[][] = [];
-          for (let l = 0; l < nightLinePoints.length - 1; l++){
-            const x1:number = Object.values(nightLinePoints[l])[0];
-            const y1:number = Object.values(nightLinePoints[l])[1];
-            const x2:number = Object.values(nightLinePoints[l+1])[0];
-            const y2:number = Object.values(nightLinePoints[l+1])[1];
-            const lineString = [fromLonLat([x1, y1]),fromLonLat([x2, y2])];
-            lineStringArray.push(lineString);
-          }
-          nightLineLayer.getSource()?.addFeature(new Feature({
-              geometry: new MultiLineString(lineStringArray),
-            }))
+    if (false) {
```

Comment:
Are we disabling crosshair sharing for now?

Reply:
yes -- lets add that back in your new PR